docs/guide/build.md

```diff
@@ -29,6 +29,16 @@ The exception is when you need to dynamically concatenate URLs on the fly. In th
 
 For advanced base path control, check out [Advanced Base Options](#advanced-base-options).
 
+### Relative base
+
+If you don't know the base path in advance, you may set a relative base path with `"base": "./"` or `"base": ""`. This will make all generated URLs to be relative from each files.
```

Comment:
```suggestion
If you don't know the base path in advance, you may set a relative base path with `"base": "./"` or `"base": ""`. This will make all generated URLs to be relative to each file.
```